compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Context.scala

```diff
@@ -114,15 +114,13 @@ class Context(val contextId: Context.ContextId) {
         Decode.decodeArchiveFromInputStream(archive.newInput)
       }.toMap
     extPackages ++= newPackages
-    defns ++= Compiler(extPackages).compilePackages(extPackages.keys)
+    defns ++= Compiler(extPackages).compilePackages(extPackages.keys, !forScenarioService)
```

Comment:
The variable name is really confusing. This whole code is only used in the scenario service. The difference is that in the IDE we run it in a mode where we do not perform validation whereas in the tests we do enable validation.

Reply:
Good point. I changes the name.